docker-compose.yml

```
@@ -136,6 +136,14 @@ services:
     restart: unless-stopped
     ports: ["4000:4000"]
 
+  osrd-images:
+    image: ghcr.io/openrailassociation/osrd-stable/osrd-images:${TAG_IMAGES-v1.0.0}
```

It feels okay to take the latest image pushed for the dev setup.

```suggestion
    image: ghcr.io/openrailassociation/osrd-stable/osrd-images:${TAG_IMAGES-latest}
```